Skewness : used to describe "Voltality"
Skewness is a metic which tells us about the location of dataset. that is, if you want to know where most of the values are concentrated on an ascending scale.

2 kinds - Positive Skew
          Negative Skew
Positive Skew - in this most of the value of dataset concentrated at the beginning of the scale.
Negative Skew - in this most ot the value of dataset concentrated at the end of the scale.


HAVING NO SKEWNESS MEANS THE DATASET IS FAIRLY SYMMETRICAL AND HAS A BELL SHAPED CURVE.

Kurtosis :
Kurtosis is a statistical measure thats used to describe "TRENDS IN CHARTS"
Kutosis for NORMAL DISTRIBUTION is "3".

TYPE -
B2 < 3, for Platykurtic Distribution
B2 = 3, for Mesokurtic Distribution
B2 > 3, for Leptokurtic Distribution

Q- Analyze 12 month interest rate of the investment to understand the risk factor for the future investment.

In [2]:
#calculate Skewness & Kurtosis in Python
import pandas as pd
import numpy as np

# calculate Skewness

# make a data frame
data = {
    "months" : ['April', 'May', 'June', 'July', 'August', 'September', 'October', 'November','December',
               'January','February', 'March'],
    "rate" : [12.05,13,11,18,10,11.5,15.08,21,6,8,13.2,7.5]
}

df = pd.DataFrame(data)

print(df)

# for calculating skewness from data frame 
# use this formula

print("The value of the skewness is")

print(df.skew())

       months   rate
0       April  12.05
1         May  13.00
2        June  11.00
3        July  18.00
4      August  10.00
5   September  11.50
6     October  15.08
7    November  21.00
8    December   6.00
9     January   8.00
10   February  13.20
11      March   7.50
The value of the skewness is
rate    0.641697
dtype: float64


In [3]:
# Calculate Kurtosis

from scipy.stats import kurtosis

print("The value of the kurtosis is")

print(df.kurtosis())

The value of the kurtosis is
rate    0.241602
dtype: float64


calculating the skewness & kurtosis of interest rate in python , we get the ositive skewed value and near from 0.
The skewness of the interest rate is 0.641697

The Kurtosis of the interest rate is 0.241602.
Kurtosis is less than 3, so this is "PLATYURTIC" distribution.


Conclusion - 
 According to the output of the data the value is positively skewed. 
Positive skewness indicates a distribution with an asymmetric tail extending toward more "Positve values".
ANd the Kurtosis is less than 3, it is a platykurtic distribution, positive kurtosis indicates a relatively "PEAKED DISTRIBUTION". and the distribution is light tails.


